MVVM Module/Source/Feed Presentation/FeedViewModel.swift

```diff
@@ -20,12 +20,17 @@ final class FeedViewModel {
 
 	var onLoadingStateChange: Observer<Bool>?
 	var onFeedLoad: Observer<[FeedImage]>?
+	var onFeedLoadError: Observer<String?>?
```

Comment:
`onFeedLoadError` could be named `onErrorStateChange` following the `onLoadingStateChange` property.
```suggestion
	var onErrorStateChange: Observer<String?>?
```